sdk/schemaregistry/Azure.Data.SchemaRegistry/src/SchemaRegistryClient.cs

```diff
@@ -138,51 +139,51 @@ await RegisterSchemaInternalAsync(groupName, schemaName, serializationType, sche
         /// Gets the schema ID associated with the schema from the SchemaRegistry service.
         /// </summary>
         /// <param name="groupName">The name of the SchemaRegistry group.</param>
-        /// <param name="schemaName">The name of the schema.</param>
+        /// <param name="name">The name of the schema.</param>
+        /// <param name="content">The string representation of the schema's content.</param>
         /// <param name="serializationType">The serialization format of the schema.</param>
-        /// <param name="schemaContent">The string representation of the schema's content.</param>
         /// <param name="cancellationToken">The cancellation token for the operation.</param>
         /// <returns>The properties of the schema, including the schema ID provided by the service.</returns>
 #pragma warning disable AZC0015 // Unexpected client method return type.
-        public virtual async ValueTask<SchemaProperties> GetSchemaIdAsync(
+        public virtual async ValueTask<SchemaProperties> GetSchemaPropertiesAsync(
             string groupName,
-            string schemaName,
+            string name,
+            string content,
             SerializationType serializationType,
-            string schemaContent,
             CancellationToken cancellationToken = default) =>
 #pragma warning restore AZC0015 // Unexpected client method return type.
-                await GetSchemaIdInternalAsync(groupName, schemaName, serializationType, schemaContent, true, cancellationToken)
+                await GetSchemaPropertiesInternalAsync(groupName, name, content, serializationType, true, cancellationToken)
                     .ConfigureAwait(false);
 
         /// <summary>
         /// Gets the schema ID associated with the schema from the SchemaRegistry service.
```

Comment:
```suggestion
        /// Gets the properties of a schema associated with the specified content from the SchemaRegistry service.
```

Reply:
I think it is fine to keep the original version here since the customer will be mostly interested in getting the schema ID so having the docstring putting the emphasis on that will help discoverability IMO.